Source/MethodCall.cs

```diff
@@ -151,39 +150,29 @@ public MethodCall(Mock mock, Condition condition, Expression originalExpression,
 			this.SetFileInfo();
 		}
 
-		public string FailMessage { get; set; }
-
-		public bool IsConditional
+		public string FailMessage
 		{
-			get { return condition != null; }
+			get => this.failMessage;
+			set => this.failMessage = value;
 		}
 
-		public bool IsVerifiable { get; set; }
+		bool IProxyCall.IsConditional => this.condition != null;
 
-		public bool Invoked { get; set; }
```

Comment:
This doesn't need to have a backing field. `Invoked` can be derived from the call count (see redefinition below).